docs/swarm/swarm-tutorial/inspect-service.md

```diff
@@ -0,0 +1,124 @@
+<!--[metadata]>
++++
+title = "Inspect the service"
+description = "Inspect the application"
+keywords = ["tutorial, cluster management, swarm"]
+[menu.main]
+identifier="inspect-application"
+parent="swarm-tutorial"
+weight=17
+advisory = "rc"
++++
+<![end-metadata]-->
+
+## Inspect a service on the Swarm
+
+When you have [deployed a service](deploy-service.md) to your Swarm, you can use
+the Docker CLI to see details about the service running in the Swarm.
+
+1. If you haven't already, open a terminal and ssh into the machine where you
+run your manager node. For example, the tutorial uses a machine named
+`manager1`.
+
+2. Run `docker service inspect --pretty SERVICE-ID` to display the details about
+a service in an easily readable format.
+
+    To see the details on the `helloworld` service:
+    ```
+    $ docker service inspect --pretty helloworld
+
+    ID:		2zs4helqu64f3k3iuwywbk49w
+    Name:		helloworld
+    Mode:		REPLICATED
+     Scale:	1
+    Placement:
+     Strategy:	SPREAD
+    UpateConfig:
+     Parallelism:	1
+    ContainerSpec:
+     Image:		alpine
+     Command:	ping docker.com
+    ```
+
+>**Tip**: To return the service details in json format, run the same command
+without the `--pretty` flag.
```

Comment:
Perhaps the `Tip` needs to be indented as well, because the code block below now shows the "fences" (backticks);

<img width="739" alt="screen shot 2016-06-13 at 15 59 17" src="https://cloud.githubusercontent.com/assets/1804568/16010098/f09f2e76-317f-11e6-97d6-92faf38d1388.png">